src/scheduler.rs

```diff
@@ -408,6 +413,7 @@ impl Scheduler {
                         } else {
                             let mut opt = CreateSelectMenuOption::default();
                             opt.label(date.format("%A %B %d"));
+                            opt.description("%v");
```

Comment:
This doesn't seem to be doing what you want.

![image](https://user-images.githubusercontent.com/525408/181819363-9a6ccd13-a4da-4871-8279-404630b340a8.png)

Reply:
Oh right, initially i thought i needed it for some length requirements and then ignored it because it turned out the rpoblem was max_values anyway